metron-analytics/metron-profiler-client/src/test/java/org/apache/metron/profiler/client/ProfileWriter.java

```diff
@@ -95,4 +102,22 @@ private void write(ProfileMeasurement m, List<Object> groups) {
     hbaseClient.mutate();
   }
 
+  public static void main(String[] args) throws Exception {
+    RowKeyBuilder rowKeyBuilder = new SaltyRowKeyBuilder();
+    ColumnBuilder columnBuilder = new ValueOnlyColumnBuilder();
+
+    Configuration config = HBaseConfiguration.create();
+    config.set("hbase.master.hostname", "node1");
+    config.set("hbase.regionserver.hostname", "node1");
+    config.set("hbase.zookeeper.quorum", "node1");
+
+    HTableProvider provider = new HTableProvider();
+    HTableInterface table = provider.getTable(config, "profiler");
+
+    long when = System.currentTimeMillis() - TimeUnit.DAYS.toMillis(2);
+    ProfileMeasurement measure = new ProfileMeasurement("profile1", "192.168.66.121", when, 15, TimeUnit.MINUTES);
```

Comment:
This doesn't actually run as part of the tests and does not consume 15 minutes.  So many times I've had to load up profile data for demo purposes.  This main() function just provides a simple means to load up a bunch of profile data in a single shot.